### PR TITLE
Restaurant page app

### PR DESCRIPTION
**Pull request by AlfredoElizarraras**
:eye:  This pull request includes:
- DOM manipulations.
- Create views from scratch in JavaScript.
- Uses of webpack as specified in the requirements.
- 3 Tabs: home, about us and menu

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Restaurant page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1"></head>
+  <body>
```
Comment (bellom) [inline dist/index.html:7]: Well done developer.... 👍 
According to the instruction on `Odin` page, we should have `<div id="content"></div>` inside the `body` tag. 
image

File: src/Menu/Menu.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      <div class="modal-body__info">
+        <p id="menuModalDescription">Modal description</p>
+        <span id="menuModalGrade" class="modal-body__grade"
+          >Modal grade</span
+        >
+      </div>
+    </div>
+  </div>
+</div>
+  `;
```
Comment (bellom) [inline src/Menu/Menu.js]: Opps 😨 
We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

image

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

File: src/about-us/aboutUs.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <div id="aboutUsId" class="about-us page d-none">
+    <article class="about-us__page">
+      <img id="about-us-image"/>
+      
+      <div class="about-us__info">
+        <h2 class="about-us__title">About us</h2>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus aliquid qui sint placeat tempora aperiam ducimus, odit cum omnis expedita minus accusantium! Debitis fugiat numquam officia eveniet nostrum ut et.</p>
+      </div>
+    </article>
+  </div>
```
Comment (bellom) [inline src/about-us/aboutUs.js]: We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

image

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

File: src/home/home.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+const HomePage = (() => {
+  const createHomePageMarkUp = () => `
+  <div id="homeId" class="home page">
+  <img id="homeImage" class="home__image"/>
+  <div class="home__info">
+    <h2 class="home__title">Restaurant Page</h2>
+    <button id="btn-home-menu" class="home__button">START EATING</button>
+  </div>
+</div>
```
Comment (bellom) [inline src/home/home.js]: We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

image

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @AlfredoElizarraras  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: dist/index.html (hunk 1 above)
Comment (AlfredoElizarraras) [reply dist/index.html:7]: Hello Oluwaseun Bello :raised_hands: 

Thanks for all your feedback, I accomplish the changes to ask for, and go to re-submit for review :+1: :roller_coaster:

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @AlfredoElizarraras kudos making changes this looks good to me, nice UI design too, this is approved :+1:  please merge and move on 

**Raphael Noriode** :mask: STAY_SAFE

**Comment by AlfredoElizarraras** [comment]
Hi @Oghenebrume50 :handshake: 

Thanks for your review, and comments :smiley: 

I'll merge then :+1:

**Pull request merged**